packages/protoplugin/src/transpile.ts

```diff
@@ -45,7 +45,7 @@ const defaultOptions: ts.CompilerOptions = {
 
   // Language and Environment
   lib: [],
-  moduleDetection: "force",
+  moduleDetection: ts.ModuleDetectionKind.Force,
```

Comment:
We refer to enum value `ts.ModuleResolutionKind.NodeJs` on line 32. The option was renamed to `Node10` and `NodeJs` is deprecated. Can you change it to `Node10`?

Reply:
Odd, my typings don't show that option or mark this as deprecated